spec/models/validations/financial_validations_spec.rb

```diff
@@ -293,6 +300,20 @@
           expect(record.errors["ecstat#{n}"]).to be_empty
         end
       end
+
+      context "when the net income is lower than the hard min for their employment status" do
+        it "adds an error" do
+          record.startdate = Time.zone.local(2022, 5, 1)
```

Comment:
I think it's worth mentioning in the context/it description this is a 22/23 log given the rest in the file are 23/24

Reply:
Ah, yes, forgot to rename it, thanks!